java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Customer/CustomerRequestBody.java

```diff
@@ -0,0 +1,13 @@
+package com.my.FoodTruckApp2.Customer;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class CustomerRequestBody {
+    private String customerFirstName;
+    private String customerLastName;
+}
```

Comment:
this is me kinda being nit picky:  can these vars just be `firstName and lastName`?